internal/upload/koji/koji.go

```diff
@@ -323,27 +322,29 @@ func (k *Koji) uploadChunk(chunk []byte, filepath, filename string, offset uint6
 			if v, ok := err.(*url.Error); ok {
 				if _, ok := v.Err.(x509.UnknownAuthorityError); ok {
 					// retry if it's a timeout
-					return strings.Contains(strings.ToLower(v.Error()), "timeout"), v
+					shouldRetry = strings.Contains(strings.ToLower(v.Error()), "timeout")
+					retErr = v
 				}
 			}
 		}
 
+		if shouldRetry {
```

Comment:
Only count if we decide to retry, not on every call of the function.

Reply:
May I ask why?